src/middleware/error/errorMiddleware.spec.ts

```diff
@@ -13,6 +13,10 @@ const errorMiddlewareCatchesErrWithStatus =
 	catchesErrWithStatus(errorMiddleware);
 
 describe('errorMiddleware', () => {
+	beforeAll(() => {
+		jest.spyOn(console, 'log').mockImplementation(() => { });
```

Comment:
These are silencing the startup prompt consoles (But they are direct console.logs). Not completely sure how they are getting printed via the middleware tests. Looking into it now.

Reply:
I think I follow; thanks!

Also I'm glad we have a project board now just because I would never have seen this PR again otherwise!

Reply:
(worth a comment next to each of these to make it clear why we're doing it, perhaps? It's the sort of thing future us might see and thing "oh we don't need those lines!")

Reply:
Yea a comment would be super nice. Something such as "We need to override the console.log here because winston has not yet overriden the native logger"